components/scream/ekat/tests/pack/pack_kokkos_tests.cpp

```diff
@@ -299,13 +299,13 @@ TEST_CASE("host_device_packs", "scream::pack")
   Kokkos::Array<size_t, num_views_per_pksize> sizes = {13, 37, 59}; // num scalars per view
   std::vector<std::vector<int> > raw_data(num_pksizes_to_test, std::vector<int>());
 
-  int total_scalars = 0; // each pksize has total_scalars of data
+  int total_flex_scalars = 0; // each pksize has total_flex_scalars of data
```

Comment:
Is it worth mentioning what "flex" or "flexible" means in this context? Is this a new concept, or do we have this elsewhere, and is it clear what it is?

Reply:
It's just to distinguish from the fixed-size views which are also tested here. I'll try to improve these comments.